mountpoint-s3-fs/scripts/network_performance.sh

```diff
@@ -1,83 +1,103 @@
 #!/bin/bash
 
-# Please ensure 'jq' and AWS CLI v2 are installed before executing this script.
-# The purpose of this script is to generate a json file contains network throughput numbers of each EC2 instance types.
-
 set -euo pipefail
 
-result_file=network_performance.json
-temp_file=/tmp/temporary
+output_file="instance_throughput.rs"
 timestamp="$(date --utc +%FT%TZ)"
-region=us-east-1
-version_number=0.0.2
-
-# Network throughput query
-query_ec2_instance_network_throughput="aws ec2 describe-instance-types
---filters \"Name=instance-type,Values=*\"
---query \"InstanceTypes[].[InstanceType, NetworkInfo.NetworkPerformance]\"
---region ${region}
---output json"
-
-# Special case handling:
-# The results for the following three instances are:
-#    dll.24xlarge  --> 4x 100 Gbps
-#    p4d.24xlarge  --> 4x 100 Gbps
-#    trn1.32xlarge --> 8x 100 Gbps
```

Comment:
Let's leave a comment explaining the special case (and one of these as an example)